vimoutliner/scripts/otl2html.py

```diff
@@ -1076,6 +1079,7 @@ def printFooter():
 
 def main():
     global showTitle
+    locale.setlocale(locale.LC_ALL, '')
```

Comment:
Please, read https://docs.python.org/2.7/library/locale.html … it does something else than you seem to think it does.